.flake8

```diff
@@ -1,3 +1,4 @@
 [flake8]
-exclude = .git, .tox, .venv, .eggs, build, dist, docs
+exclude = .git, .tox, .venv, .eggs, build, dist, docs, meteoroid/core/migrations
+max-complexity = 10
 max-line-length = 120
```

Comment:
I will change max-line-length to 100 from 120.

Reply:
Done.

Reply:
Thank your for the quick fixing over 100 codes.